src/OrchardCore.Nancy.Web/OrchardCore.Nancy.Web.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk.Web">
 
   <Import Project="..\OrchardCore.Build\Dependencies.props" />
-  <Import Project="..\OrchardCore\OrchardCore.AsBundle\OrchardCore.AsBundle.targets" />
```

Comment:
When we reference at least one bundle as a package, this targets file is embedded in the package. But when we only reference bundles as projects as here, as i remember we also need to import `OrchardCore.AsBundle.targets`.

**So, i think you need to keep this import**.

Reply:
I tried it on Orchard.Samples, referencing AsBundle only works. Can you test?

Reply:
Ok, i will try it

Reply:
@jtkech  And also, open skype

Reply:
So, if you remove this line, when you build the app, modules asset files are not copied to the `/Packages` folder.

Maybe it was working because you already had this folder.

Reply:
Ok, I see what you mean now, I will revert

Reply:
Fixed